docs/references/faq.md

```diff
@@ -295,3 +295,23 @@ To address this, use [-omit-ui in create-yxi](./ayx-plugin-cli#create-yxi).
 ## _"I don't want to rebuild UI components when installing a YXI through the CLI."_
 
 To address this, use [-omit-ui in designer-install](./ayx-plugin-cli#designer-install).
+
+## _"How do you install YXI packages into Alteryx Designer Desktop?"_
+
+There are a few options to install YXIs.
+
+1. Right-click the Designer icon and select Run As Administrator. (Important Note: If you don’t Run as Administrator, the file won’t install to the correct folder)
+   1. In Designer, go to File > Open Workflow > Browse and select the downloaded YXI file.
```

Comment:
Is this correct?  You select? I would assume if you select Open Workflow you're limited to opening workflow files.

Reply:
Correcting. Got feedback from Designer Team